eng/targets/ReferenceAssembly.targets

```diff
@@ -52,6 +52,7 @@
 
       <_RefSourceOutputPath>$([System.IO.Directory]::GetParent('$(MSBuildProjectDirectory)'))/ref/</_RefSourceOutputPath>
       <_RefSourceFileName>$(AssemblyName).$(_RefSourceFileTFM).cs</_RefSourceFileName>
+      <_ManualRefSourceFileName>$(AssemblyName).Manual.cs</_ManualRefSourceFileName>
```

Comment:
Not a big deal but you're not actually adding any *.Manual.cs files anymore i.e. the `$(AdditionalGenApiCmdOptions)` change is all you need.